specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/ConversationalAnalysis-authoring.json

```diff
@@ -0,0 +1,2742 @@
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ProjectFileExport"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "",
+            "headers": {
+              "operation-location": {
+                "description": "The location of the status API for monitoring the created job.",
+                "type": "string"
+              }
+            }
+          },
+          "400": {
+            "description": "This error can be returned if the request's parameters are incorrect meaning the required parameters are missing, malformed, or too large.\\r\\n\\r\\nThis error can be returned if the request's body is incorrect meaning the JSON is missing, malformed, or too large."
+          },
+          "401": {
+            "description": "You do not have access. \\r\\n\\r\\nReasons can include:\\r\\n\\r\\n* used endpoint subscription key, instead of authoring key\\r\\n* invalid, malformed, or empty authoring key\\r\\n* authoring key doesn't match region\\r\\n* you are not the owner or collaborator\\r\\n* invalid order of API calls."
+          },
+          "403": {
+            "description": "Total monthly key quota limit exceeded."
+          },
+          "404": {
+            "description": "The project does not exist."
+          },
+          "429": {
+            "description": "Rate limit is exceeded."
+          }
+        }
+      }
+    },
+    "/projects/{projectName}/:import": {
+      "post": {
+        "description": "Triggers a job to import a project in JSON format. If a project with the same name already exists, the data of that project is replaced.",
+        "operationId": "Projects_TriggerImportProjectJob",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "projectName",
+            "description": "The project name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "header",
+            "name": "format",
+            "description": "The format of the project to import. The currently supported format is json format.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "projectFileImportObject",
+            "description": "The project data to import.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/JsonProject"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "operation-location header for the status of the job.",
+            "headers": {
+              "operation-location": {
+                "description": "The location of the status API for monitoring the created job.",
+                "type": "string"
+              }
```

Comment:
What will happen is the request is accepted, but response is lost due to network issue?

Also it helps to provide examples, so reviewer and tooling has some idea of your request/response. https://azure.github.io/autorest/extensions/#x-ms-examples